dev/acf-export.json

```diff
@@ -81,6 +81,66 @@
                 "multiple": 1,
                 "return_format": "id",
                 "ui": 1
+            },
+            {
+                "key": "field_60413825e0ada",
+                "label": "Header Story",
+                "name": "main_site_header_story",
+                "type": "post_object",
+                "instructions": "The story to display in the header on UCF.edu (if the custom layout is active)",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "post_type": [
+                    "post"
+                ],
+                "taxonomy": "",
+                "allow_null": 1,
+                "multiple": 0,
+                "return_format": "object",
+                "ui": 1
+            },
+            {
+                "key": "field_60413bfa7e88b",
```

Comment:
Consider conditionally showing the title and subtitle override fields only when the header story field has a value to help reduce clutter and potential confusion over how to use those fields.